lib/connection.ts

```diff
@@ -127,10 +156,10 @@ class Connection {
      * @returns {Promise.<*>}
      */
   callLocalServiceMethod(methodName: string, args: any[]) {
-    if (!this.serviceMethods[methodName]) {
-      throw new Error(`Serivce method ${methodName} is not registered`);
+    if (!propertyByPath(this.serviceMethods, methodName)) {
+      throw new Error(`Service method ${methodName} is not registered`);
     }
-    return Promise.resolve(this.serviceMethods[methodName](...args));
+    return Promise.resolve((propertyByPath(this.serviceMethods, methodName) as Function)(...args));
```

Comment:
I would suggest something like this for consistency with `callLocalApi`:
```suggestion
    const method = propertyByPath<Function>(this.serviceMethods, methodName);
    if (!method) {
      throw new Error(`Service method ${methodName} is not registered`);
    }
    return Promise.resolve(method.call(this, ...args));
```